misc/config_tools/configurator/src/pages/Config.vue

```diff
@@ -177,14 +177,38 @@ export default {
     },
     deleteVM() {
       let currentVMIndex = -1;
+      let vmConfigcurrent = []
+      let postvmlist = []
+      let isserivevm = false
+      let msg=''
       this.scenario.vm.map((vmConfig, vmIndex) => {
         if (vmConfig['@id'] === this.activeVMID) {
           currentVMIndex = vmIndex
+          vmConfigcurrent = vmConfig
+        }
+        if (vmConfig['load_order'] === 'POST_LAUNCHED_VM') {
+          postvmlist.push(vmIndex)
+          console.log(postvmlist)
+        }
+      })
+      if (vmConfigcurrent['load_order'] === 'SERVICE_VM') {
+        msg = "To delete the Service VM, all Posted-Launched VMs will be deleted. Are you sure you want to delete the Service VM and all Post-Launched VMs?"
```

Comment:
Change the test when deleting the Service VM to first check if there are any post-launched VMs before writing out this message.  A better message would be:

```suggestion
        msg = "Post-launched VMs require the Service VM. If you proceed, all posted-launched VMs and their settings will also be deleted. Are you sure you want to proceed?"
```

Reply:
Thanks, will apply the change.